dev-docs/modules/yieldmoSyntheticInventoryModule.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: page_v1
+page_type: module
+title: Yieldmo Synthetic Inventory Module
+description: Yieldmo Synthetic Outstream ads
+module_code : yieldmoSyntheticInventoryModule
+display_name : Synthetic Inventory Module
+enable_download : true
+sidebarType : 1
+---
+
+# Yieldmo Synthetic Inventory Module
+
+## Overview
+
+This module enables publishers to set up Yieldmo Synthetic Outstream ads on their pages.
+
+If publishers will enable this module and provide placementId and Google Ad Manager ad unit path, this module will create a placement on the page and inject Yieldmo SDK into this placement. Publisher will then need to get a placement id from their Yieldmo account manager (accounts email) and setup corresponding ad units on the GAM ad server.
+
+## Integration
+
+Build the Yieldmo Synthetic Inventory Module into the Prebid.js package with:
+
+```
+gulp build --modules=yieldmoSyntheticInventoryModule,...
+```
+
+## Module Configuration
+
+```js
+pbjs.que.push(function() {
+    pbjs.setConfig({
+        ym_synthetic_inventory: {
```

Comment:
The code says this config is yieldmo_synthetic_inventory

Reply:
@bretg fixed, thanks